### ISSUE TITLE
skupper-service-controller's /DATA call returns binary characters in addresses for mongodb example

### ISSUE THREAD

**Issue by ErnieAllen** [skupperproject/skupper#565]
Service address with non-ascii characters are being returned in the /DATA REST endpoint.
To reproduce:
- setup the mongo db example
- curl to the console's address /DATA on each namespace
- You will see some service addresses that contain non-ascii characters

Example:
{
    "sites": [...],
    "services": [
        {
            "address": "�L\u0015$\u0017 ",
            "protocol": "tcp",
            "targets": null,
            "connections_ingress": [...]
        },
        {
            "address": " �\u001a�%! (MISSING)",
            "protocol": "tcp",
            "targets": null,
            "connections_ingress": [...]
        },
        {
            "address": "mongo-b",
            "protocol": "tcp",
            "targets": [
                {
                    "name": "mongo-b-86758f9555-2rdqg",
                    "target": "mongo-b.mongo-b",
                    "site_id": "850e8d68-5688-45bc-a4ea-d15dab0c39cf"
                }
            ],
            "connections_ingress": [...],
            "connections_egress": [...]
        },
        {
            "address": "�0��T ",
            "protocol": "tcp",
            "targets": null,
            "connections_ingress": [...]
        },
        {
            "address": "mongo-c",
            "protocol": "tcp",
            "targets": [
                {
                    "name": "mongo-c-54866cb45c-bb5kf",
                    "target": "mongo-c.mongo-c",
                    "site_id": "cdd88f2a-3ebc-4d29-8a4b-f13d5260fcb2"
                }
            ]
        },
        {
            "address": "\u0001�\u0004\u00050\u0003\u0001\u0001�\u0004\u0003\u0002\u0001�",
            "protocol": "tcp",
            "targets": null,
            "connections_egress": [...]
        }
    ]
}

The console is filtering out these services for now.

Notes/Observations:
- executing the command 'kubectl exec -it skupper-router-<xxxxxxxxxx-xxxxx> -- qdmanage query --type tcpConnection' gives the following error, but not on all namespaces:
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xf0 in position 0: invalid continuation byte
command terminated with exit code 1

- If I force the skupper-service-controller to restart, which namespaces give the above error changes

**Comment by grs** [comment]
What if you restart the routers?

**Comment by ErnieAllen** [comment]
Unfortunately, I deleted skupper from the namespaces before I saw the request to restart the routers.
However, after removing and re-installing the demo, the binary characters are no longer present in the /DATA results.
I may have installed the demo incorrectly the first time. Or maybe, as you suggested, exposing and unexposing services through Gilligan caused a problem. I'll keep trying to reproduce.
